Dockerfile.cpu

```diff
@@ -0,0 +1,21 @@
+# This vLLM Dockerfile is used to construct image that can build and run vLLM on x86 CPU platform.
+
+FROM ubuntu:22.04
+
+RUN apt-get update  -y \
+    && apt-get install -y git wget vim numactl gcc-12 g++-12 python3 python3-pip \
+    && update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 10 --slave /usr/bin/g++ g++ /usr/bin/g++-12
+
+RUN pip install --upgrade pip \
+    && pip install wheel packaging ninja setuptools>=49.4.0 numpy \
+    && pip install torch==2.1.2+cpu --index-url https://download.pytorch.org/whl/cpu
+
+COPY ./ /workspace/vllm
+
+WORKDIR /workspace/vllm
+
+RUN pip install -v -r requirements-cpu.txt
```

Comment:
Is there a way to avoid specifying the torch version here? It is best to keep it in a single source of truth, like the other `requirements-*.txt` files. Perhaps you could remove the explicit torch install and replace with 
```
RUN pip install -v -r requirements-cpu.txt --extra-index-url https://download.pytorch.org/whl/cpu
```

Reply:
Good advice, updated.